package.json

```diff
@@ -70,10 +71,29 @@
     "@seadub/danger-plugin-dependencies": "^1.0.0",
     "@seadub/danger-plugin-eslint": "^2.0.0",
     "@seadub/danger-plugin-junit": "^0.1.1",
+    "@semantic-release/changelog": "^5.0.1",
+    "@semantic-release/git": "^9.0.0",
     "danger": "^7.1.4",
     "diff": "^3.4.0",
     "eslint": "^7.5.0",
     "jake": "^8.0.12",
-    "mkdirp": "^0.5.1"
+    "mkdirp": "^0.5.1",
+    "semantic-release": "^17.1.2"
+  },
+  "release": {
+    "tagFormat": "${version}",
```

Comment:
Used when detecting previous releases, semantic-release expects `v${version}` but this repo has been using `${version}`